.github/workflows/build.yml

```diff
@@ -11,8 +11,10 @@ jobs:
     steps:
     - name: "Install dependencies"
       run: |
+        # We need SDL 2.0.5+
+        sudo tee /etc/apt/sources.list.d/bionic.list <<<'deb http://azure.archive.ubuntu.com/ubuntu bionic main restricted universe multiverse'
```

Comment:
This is normally a cardinal sin in Debian-likes - and looks like in [the logs](https://github.com/withinwheels/BrogueCE/runs/1700275892?check_suite_focus=true#step:2:156) that we install a whole bunch of core system stuff from bionic, potentially losing 16.04 compat anyway. If we want this as a temp fix then we probably just bump the VM to ubuntu-18.04.

On this subject, I believe we can easily swap out the problematic SDL_CreateRGBSurfaceWithFormat function with a combo of SDL_CreateRGBSurface and [SDL_PixelFormatEnumToMasks](https://wiki.libsdl.org/SDL_PixelFormatEnumToMasks), which are present in SDL <2.0.5. I'll have a look

Reply:
Yea, I had mixed feelings on this.  If you're comfortable moving to bionic then that may be the simplest solution.  We could also consider bringing SDL in some other way, but none of those options look especially friendly, fast or easy.